build.sh

```diff
@@ -333,8 +337,7 @@ if buildDefault || hasArg pylibcugraph || hasArg all; then
                    -j${PARALLEL_LEVEL:-1}
             cd -
         fi
-        SKBUILD_CONFIGURE_OPTIONS="${SKBUILD_EXTRA_CMAKE_ARGS} -DUSE_CUGRAPH_OPS=${BUILD_WITH_CUGRAPHOPS}" \
-            SKBUILD_BUILD_OPTIONS="-j${PARALLEL_LEVEL}" \
+        SKBUILD_CMAKE_ARGS="${SKBUILD_EXTRA_CMAKE_ARGS};-DUSE_CUGRAPH_OPS=${BUILD_WITH_CUGRAPHOPS}" \
```

Comment:
I can't comment on lines too far outside the change, but can the `FIXME` just above (starting on line 324) and below (starting on line 350) be removed now?